pkgs/development/python-modules/playwright/default.nix

```diff
@@ -0,0 +1,223 @@
+{ lib
+, stdenv
+, autoPatchelfHook
+, callPackage
+, fetchFromGitHub
+, fetchurl
+, makeWrapper
+, runCommand
+, unzip
+, nodejs
+
+# Python dependencies
+, python
+, buildPythonPackage
+, pythonOlder
+, autobahn
+, greenlet
+, objgraph
+, pillow
+, pixelmatch
+, pyee
+, pyopenssl
+, requests
+, service-identity
+, websockets
+
+# For building the browser bundle
+, cacert
+, chromium
+, ffmpeg
+, makeFontsConf
+}:
+
+let
+  inherit (stdenv.hostPlatform) system;
+
+  driverVersion = "1.25.0";
+
+  driver = let
+    suffix = {
+      x86_64-linux = "linux";
+      aarch64-linux = "linux-arm64";
+      x86_64-darwin = "mac";
+      aarch64-darwin = "mac-arm64";
+    }.${system};
+    filename = "playwright-${driverVersion}-${suffix}.zip";
+  in stdenv.mkDerivation {
+    pname = "playwright-driver";
+    version = driverVersion;
+
+    src = fetchurl {
+      url = "https://playwright.azureedge.net/builds/driver/${filename}";
+      sha256 = {
+        x86_64-linux = "1nah45q9mihzkz2hqx6dwn7ac88vc22m5zspr23hc7dd1q4ll4ys";
+        aarch64-linux = "0wg4z2gb2fhvv4c2gjm6ncvbg9bkaavzq76x1bmm4lsnhjz312hp";
+        x86_64-darwin = "1nd6ll6dj0aqic3hh65y0br051ckbcap8spqbvzm19g8dn7vgv7m";
+        aarch64-darwin = "08qrzahmdhdv8flfjb58m27dqihq8jc4gwix6knc71209a9x8b83";
+      }.${system};
+    };
+    sourceRoot = ".";
+
+    nativeBuildInputs = [ unzip ];
+    buildInputs = [ nodejs ];
+
+    postPatch = ''
+      # Use Nix's NodeJS instead of the bundled one.
+      substituteInPlace playwright.sh --replace '"$SCRIPT_PATH/node"' '"${nodejs}/bin/node"'
+      rm node
+
+      # Hard-code the script path to be our $out directory. The alternative would be to add
+      # coreutils to the closure and use realpath like the upstream script does.
+      substituteInPlace playwright.sh --replace \
+        'SCRIPT_PATH="$(cd "$(dirname "$0")" ; pwd -P)"' \
+        "SCRIPT_PATH=$out"
+
+      patchShebangs playwright.sh
+      patchShebangs package/bin/*.sh
+    '';
+
+    dontBuild = true;
+
+    installPhase = ''
+      mkdir -p $out/bin
+      mv playwright.sh package $out/
```

Comment:
This shouldn't be in $out/

Reply:
Why? The drivers package is basically the extracted compiled drivers from upstream, patched to use Nix store paths for the Node interpreter.